src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/ServerCertificateTest.cs

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.Net.Tests;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.WinHttpHandlerFunctional.Tests
+{
+    public class ServerCertificateTest
+    {
+        private readonly ITestOutputHelper _output;
+        private readonly ValidationCallbackHistory _validationCallbackHistory;
+
+        public ServerCertificateTest(ITestOutputHelper output)
+        {
+            _output = output;
+            _validationCallbackHistory = new ValidationCallbackHistory();
+        }
+
+        [Fact]
+        public async Task NoCallback_ValidCertificate_CallbackNotCalled()
+        {
+            var handler = new WinHttpHandler();
```

Comment:
WinHttpHandler is Disposable. 
A test where it's not properly disposed is interesting. If that's the case, please add a top-of-file general comment explaining that is by intent.

If not, it can easily be included in the using chain:

```C#
using (var handler = new WinHttpHandler())
using (var client = new HttpClient(handler))
{ 
...
}
```